doc/_scripts/redirects.py

```diff
@@ -18,15 +18,33 @@
     ('boards/arduino/uno_r4_minima/doc/index', 'boards/arduino/uno_r4/doc/index'),
     ('boards/x86/ehl_crb/doc/index', 'boards/x86/intel_ehl/doc/index'),
     ('boards/x86/rpl_crb/doc/index', 'boards/x86/intel_rpl/doc/index'),
+    ('connectivity/bluetooth/audio', 'connectivity/bluetooth/api/audio/audio'),
+    ('connectivity/bluetooth/bap', 'connectivity/bluetooth/api/audio/bap'),
     ('connectivity/bluetooth/bluetooth-audio-arch', 'connectivity/bluetooth/bluetooth-le-audio-arch'),
+    ('connectivity/bluetooth/bluetooth-le-audio-arch', 'connectivity/bluetooth/api/audio/bluetooth-le-audio-arch'),
+    ('connectivity/bluetooth/cap', 'connectivity/bluetooth/api/audio/cap'),
+    ('connectivity/bluetooth/coordinated_sets', 'connectivity/bluetooth/api/audio/coordinated_sets'),
     ('connectivity/bluetooth/dis-pics', 'connectivity/bluetooth/ics/dis'),
     ('connectivity/bluetooth/gap-pics', 'connectivity/bluetooth/ics/gap'),
     ('connectivity/bluetooth/gatt-pics', 'connectivity/bluetooth/ics/gatt'),
     ('connectivity/bluetooth/l2cap-pics', 'connectivity/bluetooth/ics/l2cap'),
+    ('connectivity/bluetooth/media', 'connectivity/bluetooth/api/audio/media'),
     ('connectivity/bluetooth/mesh-pics', 'connectivity/bluetooth/ics/mesh'),
+    ('connectivity/bluetooth/microphone', 'connectivity/bluetooth/api/audio/microphone'),
     ('connectivity/bluetooth/overview', 'connectivity/bluetooth/features'),
     ('connectivity/bluetooth/rfcomm-pics', 'connectivity/bluetooth/ics/rfcomm'),
+    ('connectivity/bluetooth/shell/bap', 'connectivity/bluetooth/api/audio/shell/bap'),
+    ('connectivity/bluetooth/shell/bap_broadcast_assistant', 'connectivity/bluetooth/api/audio/shell/bap_broadcast_assistant'),
+    ('connectivity/bluetooth/shell/bap_scan_delegator', 'connectivity/bluetooth/api/audio/shell/bap_scan_delegator'),
+    ('connectivity/bluetooth/shell/cap', 'connectivity/bluetooth/api/audio/shell/cap'),
+    ('connectivity/bluetooth/shell/ccp', 'connectivity/bluetooth/api/audio/shell/ccp'),
+    ('connectivity/bluetooth/shell/csip', 'connectivity/bluetooth/api/audio/shell/csip'),
+    ('connectivity/bluetooth/shell/gmap', 'connectivity/bluetooth/api/audio/shell/gmap'),
+    ('connectivity/bluetooth/shell/mcp', 'connectivity/bluetooth/api/audio/shell/mcp'),
+    ('connectivity/bluetooth/shell/pbp', 'connectivity/bluetooth/api/audio/shell/pbp'),
+    ('connectivity/bluetooth/shell/tmap', 'connectivity/bluetooth/api/audio/shell/tmap'),
```

Comment:
Alternative path for the shell files would be `connectivity/bluetooth/shell/audio`

Reply:
> but it should be done in a separate PR where we also move the mesh-shell description

If it makes more sense to have in `connectivity/bluetooth/shell/audio` then it's better to fix ASAP, rather than in a later PR as that would just create more redirections.